avm/res/event-hub/namespace/authorization-rule/main.json

```diff
@@ -4,8 +4,8 @@
   "metadata": {
     "_generator": {
       "name": "bicep",
-      "version": "0.33.93.31351",
-      "templateHash": "10622466301065060150"
+      "version": "0.32.4.45862",
```

Comment:
Just a heads up to update the bicep cli version before compiling

Reply:
recompiled with updated version